nextflow.config

```
@@ -135,17 +137,23 @@ profiles {
         docker.runOptions       = '-u $(id -u):$(id -g)'
     }
     arm {
-        docker.runOptions       = '-u $(id -u):$(id -g) --platform=linux/amd64'
+        if (params.use_gpu) {
+            docker.runOptions = '-u $(id -u):$(id -g) --platform=linux/amd64 --gpus all'
+        } else {
+            docker.runOptions = '-u $(id -u):$(id -g) --platform=linux/amd64'
+        }
     }
     singularity {
         singularity.enabled     = true
         singularity.autoMounts  = true
+        if (params.use_gpu) { singularity.runOptions = '--nv' }
```

```suggestion
        singularity.runOptions = params.use_gpu ? '--nv' : singularity.runOptions
```
Also maybe add it for `apptainer`: `apptainer.runOptions = params.use_gpu ? '--nv' : apptainer.runOptions`